sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -2391,6 +2391,17 @@ public User getRandomRegionContactResponsible(Region region) {
 		return userService.getRandomRegionUser(region, UserRight.CONTACT_RESPONSIBLE);
 	}
 
+	@Override
+	@RightsAllowed(UserRight._PERSON_EDIT)
```

Comment:
I was unsure about that, too. The method is currently only used in methods that require the CONTACT_CREATE permission. However, the method itself only changes information in the person. I will ask @MartinWahnschaffe.

Reply:
I thing PERSON_EDIT is fine, since this is the data that is changed here. 
But: I'd change the whole method a bit, like described here: https://github.com/hzi-braunschweig/SORMAS-Project/pull/10682#discussion_r1004436695